XCGLogger/Library/XCGLogger/XCGLogger.swift

```diff
@@ -167,6 +167,7 @@ public class XCGConsoleLogDestination: XCGBaseLogDestination {
 
     // MARK: - Misc Methods
     public override func output(logDetails: XCGLogDetails, text: String) {
+        let queue = dispatch_get_global_queue(QOS_CLASS_BACKGROUND, 0)
```

Comment:
isn't `QOS_CLASS_BACKGROUND` a concurrent queue? Usually this leads to a mess when you log from multiple queues, since the log messages tend to overlap